LuaSkin/LuaSkin/Skin.m

```diff
@@ -1065,7 +1065,7 @@ - (NSSize)tableToSizeAtIndex:(int)idx {
 
 #pragma mark - Support Functions
 
-// maxn   returns the largest integer key in the table
+// maxNatIndex returns the largest integer key in the table
 - (lua_Integer)maxNatIndex:(int)idx {
```

Comment:
I'm not terribly confident about this. The whitespace is pretty odd.

It appears that the function means `max-n-at-index`. I'm not sure if the name has some magical lua mappings.